plugins/modules/swim_intent.py

```diff
@@ -81,19 +81,20 @@
                 elements: dict
                 suboptions:
                   application_type:
-                    description: Swim Import Via Url's applicationType.
+                    description: Optional parameter indicating the type of application with permitted values(WLC, LINUX, FILREWALL, WINDOWS,
+                      LOADBALANCER, THIRDPARTY etc) applicable only for third party image types.
                     type: str
                   image_family:
-                    description: Swim Import Via Url's imageFamily.
+                    description: The name of image family applicable only in case of third party images upload (Optional).
                     type: str
                   source_url:
-                    description: Swim Import Image Via Url.
+                    description: Required parameter for importing swim image via remote url.
                     type: str
                   is_third_party:
-                    description: ThirdParty flag.
+                    description: Set the boolean value if image is uploaded from third party (Optional).
                     type: bool
                   vendor:
-                    description: Swim Import Via Url's vendor.
+                    description: Name of vendor applicable only for third party image types (Optional).
```

Comment:
The name of the vendor, that applies only to third-party image types when importing via URL.

Reply:
added.